zomes/profile/src/error.rs

```diff
@@ -18,6 +18,6 @@ pub type ProfileResult<T> = Result<T, ProfileError>;
 
 impl From<ProfileError> for WasmError {
     fn from(c: ProfileError) -> Self {
-        WasmError::Guest(c.to_string())
+        wasm_error!(WasmErrorInner::Guest(c.to_string()))
```

Comment:
Feels like we should have a macro that just takes a string and produces a `WasmErrorInner::Guest` implicity, what do you think @thedavidmeister?

Reply:
@maackle  oh yeah seems fair